### DISCUSSION TITLE
Move repo to bootstrap-vue-3 organization

### DISCUSSION THREAD

**Discussion by cdmoro** [cdmoro/bootstrap-vue-3#290] [Ideas]
I've created this organization some time ago. I propose to move this repo to https://github.com/bootstrap-vue-3

**Comment by mariusa** [comment]
I know there are talks with `bootstrap-vue` new maintainer and their port to Vue 3. Would you please discuss with him/them concrete plans about bootstrap-vue for Vue 3 and BS 5?

**Comment by micene09** [comment]
Personally, I think that this amazing work you started, is the BootstrapVue rewrite that everyone was waiting for.

I'm checking like everyday the status quo of the BootstrapVue lib, because in my company we have 4 big portals (about 5 years of codebase) using it and we all want to upgrade to Vue 3 and bootstrap 5.
What I see from the BootstrapVue community is that the compatibility layer was almost ready but...Russia was invaded and I hope that xanf (author of the compat layer, from Kharkiev) stay safe.

Yesterday I had a chat with via Discord with the BootstrapVue Discord channel Admin and...without xanf and pi0 (the original author, having full priority on Nuxt 3), they feel like the project is almost dead.
So many people are switching to React, or are just using other libraries...this is a signal.

What I propose is:
1. Let's move this to the BootstrapVue org
2. Let's define guide lines, a more detailed and strategic project roadmap and clear goals (of implementations), so that people are more guided on what to do on their pull requests

I'm just trying to help...and I'm open for collaboration about board coordination and goals definition.
